struct.c

```diff
@@ -1154,19 +1178,28 @@ invalid_struct_pos(VALUE s, VALUE idx)
 
 /*
  *  call-seq:
- *     struct[member]   -> object
- *     struct[index]    -> object
+ *    struct[name] -> object
+ *    struct[n] -> object
```

Comment:
I would use a single line with `key` instead of `name` and `n`.  From a caller's perspective, the two lines are the same.  Same change for `[]=`

Reply:
These two forms seem different to me.

More importantly,  replacing name and n with key would force some clumsiness into the discussion:
- +key+th or something perhaps worse.
- Raises NameError if +key+ is not an integer or a member name.
- Raises IndexError if +key+ is an integer that is out of range.

I'd prefer to leave stet.

Reply:
Since this is just a preference on my part, I'm fine leaving as-is.